### ISSUE TITLE
ITS alerts calendar accessibility issues

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9896]
# Motivation

Siteimprove is reporting some accessibility issues with the Full Calendar implementation of the ITS alerts calendar. Some of these are compliance issues that should be resolved prior to the deadline.

# Additional Info

In https://github.com/uiowa/uiowa/issues/9790, we switched to using the full calendar module in order to better position ourselves for D11 and for future accessibility fixes.

Previous work in https://github.com/uiowa/uiowa/issues/9365#issuecomment-4336350264, also lead us to believe that the "Scrollable element is not keyboard accesible" issue may be something we have introduced since it wasn't observed outside of SiteNow implementations.

If we cannot reach compliance with the calendar display, an alternate accessible list display that meets compliance may be necessary.

# Siteimprove Info

**ARIA attribute unsupported or prohibited** - ARIA

Technical doc
https://alfa.siteimprove.com/rules/sia-r18

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/994335/A11y/Page?pageId=28396872889&impmd=264ED37B65FA6186129334A508391970#/sia-r18/failed

**Table header cell is missing a header role** - WCAG A Compliance

Technical doc
https://alfa.siteimprove.com/rules/sia-r76

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/994335/A11y/Page?pageId=28396872889&impmd=264ED37B65FA6186129334A508391970#/sia-r76/failed/VzVCNld0WjM=;MGI0ZjkzNjg=

**Scrollable element is not keyboard accessible** - WCAG A Compliance

Technical doc
https://alfa.siteimprove.com/rules/sia-r84

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/994335/A11y/Page?pageId=28396872889&impmd=264ED37B65FA6186129334A508391970#/sia-r84/failed/cTdIa2dhM3U=;YmMyM2U2MzM=
